scripts/scaletest.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
+
+if [[ $# -lt 2 ]]; then
+	echo "Usage: $0 <name> <scenario> <num_workspaces>"
+fi
+
+[[ -n ${VERBOSE:-} ]] && set -x
+set -euo pipefail
+
+SCALETEST_NAME="$1"
+SCALETEST_SCENARIO="$2"
+SCALETEST_NUM_WORKSPACES="$3"
+SCALETEST_PROJECT="${SCALETEST_PROJECT:-}"
+PROJECT_ROOT="$(git rev-parse --show-toplevel)"
+SCALETEST_SCENARIO_VARS="${PROJECT_ROOT}/scaletest/terraform/scenario-${SCALETEST_SCENARIO}.tfvars"
+SCALETEST_SECRETS="${PROJECT_ROOT}/scaletest/terraform/secrets.tfvars"
+SCALETEST_SECRETS_TEMPLATE="${PROJECT_ROOT}/scaletest/terraform/secrets.tfvars.tpl"
+SCALETEST_SKIP_CLEANUP="${SCALETEST_SKIP_CLEANUP:-}"
+
+if [[ "${SCALETEST_SKIP_CLEANUP}" == "true" ]]; then
+	echo "WARNING: you told me to not clean up after myself, so this is now your job!"
+fi
+
+if [[ -z "${SCALETEST_PROJECT}" ]]; then
+	echo "Environment variable SCALETEST_PROJECT not set. Please set it and try again."
+	exit 1
+fi
+
+if [[ ! -f "${SCALETEST_SCENARIO_VARS}" ]]; then
+	echo "No definition for scenario ${SCALETEST_SCENARIO} exists. Please create it and try again"
+	exit 1
+fi
+
+echo "Writing scaletest secrets to file."
+SCALETEST_NAME="${SCALETEST_NAME}" envsubst <"${SCALETEST_SECRETS_TEMPLATE}" >"${SCALETEST_SECRETS}"
```

Comment:
seems like the user has to have some environment variables set for this to work --- can we document somewhere like a README?